app/views/application/info_pair_programming.html.slim

```diff
@@ -0,0 +1,7 @@
+= render_markdown(t('programming_groups.new.info_pair_programming'))
+= render_markdown(t('programming_groups.new.info_study'))
+== t('exercises.implement.pair_programming_feedback', url: 'https://etherpad.xopic.de/p/openHPI_PairProgrammingFeedback')
+
+- content_for :modal_footer do
+  button.btn.btn-primary data-bs-dismiss= 'modal' onclick= 'CodeOceanEditor.setStoredViewedPPInfo(true);'
```

Comment:
For me, the `onclick` handler is not working (which is inline with my expectation) due to the [CSP](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP):

`
Refused to execute inline event handler because it violates the following Content Security Policy directive: "script-src 'self' 'report-sample'". Either the 'unsafe-inline' keyword, a hash ('sha256-...'), or a nonce ('nonce-...') is required to enable inline execution. Note that hashes do not apply to event handlers, style attributes and javascript: navigations unless the 'unsafe-hashes' keyword is present.
`

Since we won't change the CSP, you'll need to find another solution.